src/man/firejail.1.in

```diff
@@ -2172,7 +2172,7 @@ $ pwd
 
 .TP
 \fB\-\-private-dev
-Create a new /dev directory. Only disc, dri, dvb, hidraw, null, full, zero, tty, pts, ptmx, random, snd, urandom, video, log, shm and usb devices are available.
+Create a new /dev directory. Only disc, dri, dvb, full, hidraw, log, null, ptmx, pts, random, shm, snd, tty, urandom, video, usb and zero devices are available.
```

Comment:
This also applies to private-dev in firejail-profile.5.in.

Note: These changes likely conflict with #6390.

Reply:
Argh, I forgot about #6390. Probably best to drop this PR and request sorting changes on the relevant parts of #6390. What do you advise?

Reply:
> Argh, I forgot about #6390. Probably best to drop this PR and request sorting
> changes on the relevant parts of #6390. What do you advise?

I was going to say to wait for #6390 since it was already reviewed and
approved, but I also wanted to improve the private-dev lines as they are the
only ones that are very long in the #6390 diff.

So since this is already open let's just keep going with this PR and I'll
rebase and fix the conflicts in #6390 later.